src/main/java/org/corpus_tools/peppermodules/rstModules/RSTImporterProperties.java

```diff
@@ -72,10 +72,10 @@ public class RSTImporterProperties extends PepperModuleProperties {
 	public final static String PROP_SEGMENT_SEPARATOR = PREFIX + "segmentSeparator";
 
 	/**
-	 * Name of the property to specify the name of the SLayer that contains
-	 * signals.
+	 * yes/no property which adds an is_signaled annotation to edges indicating whether it has an associated
+	 * signal when set to yes
 	 */
-	public final static String PROP_SIGNALS_LAYER_NAME = PREFIX + "signalsLayerName";

```

Comment:
What is the reasoning behind removing this property? This seems like a breaking change, and we would need to update the version number to a major release, but also consider the impact the removal has on existing workflows.

Reply:
In the course of developing this PR, we discovered a few things:

1. There is no reason why we need a layer for signals--I added this property kind of thoughtlessly during my first bit of work on this module and in retrospect, it has no purpose.
2. Including this layer can actually cause some headaches downstream in ANNIS. I can't remember details at this point, but I think including signals in both the "rst" and "signals" (or whatever) layer led to some annoyances.

In light of this, combined with the fact that we're fairly sure (I think @amir-zeldes would agree) that nobody is using this module for the signals extension to the RST module, I think it'd make everyone's life easier if we got rid of this property, as practically nobody would be affected by the breaking change, and leaving the property in might lead to confusion in users who might think this property is actually important.